package.json

```diff
@@ -2,8 +2,8 @@
   "name": "react-intl-native",
   "version": "2.1.1",
   "description": "Convenience components for react-intl's `format*` API in React Native",
-  "main": "dist/react-intl-native.js",
-  "jsnext:main": "dist/react-intl-native.es2015.js",
+  "main": "src/index.js",
+  "jsnext:main": "src/index.js",
```

Comment:
`jsnext:main` is deprecated. I don't think we should use it. If we need an ES2015 version to be explicitly defined, it should be under `module`.